tools/update-webpage.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -ne 1 ]; then
+  echo "Please, specify your gh-pages clone directory: update-webpage <gh-pages clone dir>"
+  exit 1
+fi
+
+gh_pages_dir=$1
+docs_dir="../docs"
+
+GETTING_STARTED_MD="GETTING-STARTED.md"
+API_REFERENCE_MD="API-REFERENCE.md"
+API_EXAMPLES_MD="API-EXAMPLE.md"
+INTERNALS_MD="INTERNALS.md"
+
+declare -A titles
+declare -A permalink
+declare -A order
+
+titles[$GETTING_STARTED_MD]="Getting Started"
+titles[$API_REFERENCE_MD]="API Reference"
+titles[$API_EXAMPLES_MD]="API Examples"
+titles[$INTERNALS_MD]="Internals"
+
+permalink[$GETTING_STARTED_MD]='/getting-started/'
+permalink[$API_REFERENCE_MD]='/api-reference/'
+permalink[$API_EXAMPLES_MD]='/api-example/'
+permalink[$INTERNALS_MD]='/internals/'
+
+order[$GETTING_STARTED_MD]='01.'
+order[$API_REFERENCE_MD]='02.'
+order[$API_EXAMPLES_MD]='03.'
+order[$INTERNALS_MD]='04.'
```

Comment:
I see. In that case, can't we just have the ordering by default for the input filenames? So that we don't need to hardcode here. My main problem is that with this if I add a new md file I'll need to add everything here also. It would be really great that just by adding the md file this script still works. 

The premalink could be calculated from the md file name. Only the title can't be extracted from the md file name.